pages/common/zsh.md

```diff
@@ -1,25 +1,33 @@
 # zsh
 
 > Z SHell, a Bash-compatible command-line interpreter.
-> See also `histexpand` for history expansion.
+> See also: `bash`, `histexpand` (history expansion).
 > More information: <https://www.zsh.org>.
 
 - Start an interactive shell session:
 
 `zsh`
 
-- Execute a command and then exit:
+- Execute specific [c]ommands:
 
-`zsh -c "{{command}}"`
+`zsh -c "{{echo 'zsh is executed'}}"`
 
-- Execute a script:
+- Execute a specific script:
 
 `zsh {{path/to/script.zsh}}`
 
-- Execute a script, printing each command before executing it:
+- Check a specific script for syntax errors:
+
+`zsh --no-exec {{path/to/script.zsh}}`
+
+- Execute specific commands from stdin:
+
+`{{echo "echo 'zsh is executed'"}} | zsh`
```

Comment:
```suggestion
`{{printf "echo 'zsh is executed'"}} | zsh`
```

Reply:
It looks nicer because there are no repeated commands (`echo`). On the other hand the current variant with two `echo` is simpler. 🤔

Reply:
I prefer the current choice.